data/json/items/generic.json

```
@@ -2953,6 +2953,20 @@
     "use_action": [ "CRAFT" ],
     "flags": [ "NO_SALVAGE", "TRADER_AVOID", "CAN_HAVE_CHARGES" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "disassembly",
+    "symbol": "%",
+    "color": "white",
+    "name": { "str": "in progress disassembly" },
+    "description": "This is an in progress disassembly.",
```

```suggestion
    "name": { "str": "in-progress disassembly" },
    "description": "This is an in-progress disassembly.",
```
Could also be "in-process" or "incomplete".

---

The reason it is the way it is is because that's how the existing in progress crafting item is named.  I agree that it looks nicer with a hyphen but don't want to add changing the crafting object name to this PR.  A PR to correct it for both once this merges seems like a good idea.